### DISCUSSION TITLE
4.13.0 download labeled as virus by Chrome and Edge

### DISCUSSION THREAD

**Discussion by jsartorisd127** [DistroAV/DistroAV#929] [General]
Looking forward to the latest versions.  Just finished installing a bunch of new BirdDog equipment and working on updating software, but my download is being blocked.  

the Sha-256 checksum does match the release notes.
obs-ndi-4.13.0-windows-x64-Installer.exe: 101dc8a7e4e84dccf2f89d3a56eeb4c18b9fe83eff54c3a23f2d104b3545e0c5

Defender finds : Trojan:Script/Phonzy.C!ml

**Comment by Trouffman** [comment]
When I download from the release page (in Chrome) : https://github.com/obs-ndi/obs-ndi/releases/tag/4.13.1
`obs-ndi-4.13.1-windows-x64-Installer.exe: a73f34baea9b3739fa501b7311cd00cd24ad243755e6081c24e7996141b671ea`

And then check the SHA256 hash of the file  I get the same as the release:
`SHA256 hash of .\obs-ndi-4.13.1-windows-x64-Installer.exe: a73f34baea9b3739fa501b7311cd00cd24ad243755e6081c24e7996141b671ea`

Please download this from the official release page.

This is the commadn to check on Windows (Powershell)
`certUtil -hashfile '.\obs-ndi-4.13.1-windows-x64-Installer.exe' SHA256`
